assets/src/edit-story/components/canvas/page.js

```diff
@@ -23,158 +23,83 @@ const Background = styled.div.attrs( { className: 'container' } )`
 	height: 100%;
 `;
 
-const Selection = styled.div`
-	z-index: 2;
-	left: ${ ( { x } ) => `${ x }px` };
-	top: ${ ( { y } ) => `${ y }px` };
-	width: ${ ( { width } ) => `${ width }px` };
-	height: ${ ( { height } ) => `${ height }px` };
-	transform: ${ ( { rotationAngle } ) => `rotate(${ rotationAngle }deg)` };
-	position: absolute;
-`;
-
 const Element = styled.div`
 	cursor: pointer;
 	user-select: none;
 `;
 
 function Page() {
-	const [ targetEl, setTargetEl ] = useState( null );
-
-	const {
-		state: { currentPage, hasSelection, selectedElements },
-		actions: { clearSelection, selectElementById, setPropertiesOnSelectedElements, toggleElementIdInSelection },
-	} = useStory();
 	const {
 		actions: { setBackgroundClickHandler },
 	} = useCanvas();
-	const handleSelectElement = useCallback( ( id ) => ( evt ) => {
+
+	const {
+		state: { currentPage, selectedElements },
+		actions: { clearSelection, selectElementById, toggleElementIdInSelection },
+	} = useStory();
+
+	const [ targetEl, setTargetEl ] = useState( null );
+	const [ pushEvent, setPushEvent ] = useState( null );
+
+	useEffect( () => {
+		setBackgroundClickHandler( () => clearSelection() );
+	}, [ setBackgroundClickHandler, clearSelection ] );
+
+	const handleSelectElement = useCallback( ( elId, evt ) => {
 		if ( evt.metaKey ) {
-			toggleElementIdInSelection( id );
+			toggleElementIdInSelection( elId );
 		} else {
-			selectElementById( id );
+			selectElementById( elId );
 		}
 		evt.stopPropagation();
-	}, [ toggleElementIdInSelection, selectElementById ] );
-	const selectionProps = hasSelection ? getUnionSelection( selectedElements, 0 ) : {};
-	useEffect( () => {
-		setBackgroundClickHandler( () => clearSelection() );
-	}, [ setBackgroundClickHandler, clearSelection ] );
 
-	let displayMoveable = false;
-	const frame = {
-		translate: [ 0, 0 ],
-		rotate: selectionProps.rotationAngle,
-	};
+		// @todo That's not necessary for multi-selection.
```

Comment:
Hmm, to think of it, it's actually relevant for multi-selection anyway as well since even if there's multi-selection previously, then it should be able to start dragging a non-selected element.

Reply:
Do you mean how should onPointerDown -> drag behave with multi-selection? I think that case is quite obscure and either way is ok in that case.

Reply:
@miina that's correct. If you have elements 1-5 selected in multi-selection, and start dragging on unselected element 6, the multi-selection should go away, element 6 should be selected and dragged.

Reply:
@pbakaus, what if the user holds down the meta key for multi-selecting elements (be that <kbd>shift</kbd>, <kbd>ctrl</kbd> or <kbd>cmd</kbd>) when click-dragging the 6th element?

Reply:
Note: In #3909, the idea is to ignore `onPointerDown` if a meta key is held down and add the element to the selection. Perhaps that's a good discussion for that PR.

Reply:
yes, I think ignoring the click then is likely the most logical UX. Let's go with that for now, we can revisit if it feels strange.